store/postgres/tests/chain_head.rs

```diff
@@ -35,18 +35,18 @@ where
         Err(err) => err.into_inner(),
     };
 
-    runtime
-        .block_on(future::lazy(move || {
+    let _ = runtime
```

Comment:
Great catch! For some tests this might not be an issue because they run entirely when `test` is called, but this really is wrong, fixed it by adding a `.await`.